<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>960</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,5">
    <item>
     <widget class="QWidget" name="widget_button" native="true">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(240, 255, 185);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_Network" native="true">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="1,7">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;楷体&quot;;
background-color: rgb(170, 170, 85);
color: rgb(0, 0, 255);</string>
            </property>
            <property name="text">
             <string>测试区域</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QPushButton" name="pushButton_test">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>test</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_recommend">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <property name="text">
                <string>推荐</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_Local" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,5">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_Localtext">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;楷体&quot;;
background-color: rgb(170, 170, 85);
color: rgb(0, 0, 255);</string>
            </property>
            <property name="text">
             <string>本地音乐</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_llpush" native="true">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,5,5,0,0">
             <item>
              <widget class="QPushButton" name="pushButton_Favorites">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>收藏夹</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_file">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <property name="text">
                <string>本地音乐</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_record">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <property name="text">
                <string>播放记录</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_return">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>50</height>
                </size>
               </property>
               <property name="text">
                <string>返回</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_main" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_7" stretch="5,1">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_show" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,1,2">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <item>
           <widget class="QTabWidget" name="bigtabs">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab_image">
             <attribute name="title">
              <string>主界面</string>
             </attribute>
             <widget class="QWidget" name="widget_info" native="true">
              <property name="geometry">
               <rect>
                <x>-20</x>
                <y>0</y>
                <width>1026</width>
                <height>542</height>
               </rect>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image: url(:/image/music_background.jpg);</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="tab_log">
             <attribute name="title">
              <string>记录</string>
             </attribute>
             <widget class="QWidget" name="widget" native="true">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>1011</width>
                <height>542</height>
               </rect>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="25,18,18">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QListWidget" name="listWidget_name">
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font: 11pt &quot;楷体&quot;;
color: rgb(0, 0, 255);
background-color: rgb(255, 255, 127);</string>
                 </property>
                 <item>
                  <property name="text">
                   <string>名称</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <widget class="QListWidget" name="listWidget_starttime">
                 <property name="styleSheet">
                  <string notr="true">font: 11pt &quot;楷体&quot;;
color: rgb(0, 0, 255);
background-color: rgb(255, 255, 127);</string>
                 </property>
                 <item>
                  <property name="text">
                   <string>开始时间</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <widget class="QListWidget" name="listWidget_endtime">
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font: 11pt &quot;楷体&quot;;
color: rgb(0, 0, 255);
background-color: rgb(255, 255, 127);</string>
                 </property>
                 <item>
                  <property name="text">
                   <string>结束时间</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_probressbar" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>1050</width>
              <height>25</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="4,1">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QSlider" name="horizontalSlider_time">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_time" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="5,1,5">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_currenttime">
                  <property name="text">
                   <string>00:00</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_time_separator">
                  <property name="text">
                   <string>-</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_totaltime">
                  <property name="text">
                   <string>00:00</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_textnote" native="true">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>100</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_filename">
               <property name="styleSheet">
                <string notr="true">font: 75 11pt &quot;Arial&quot;;
color: rgb(255, 0, 255);</string>
               </property>
               <property name="text">
                <string>无音乐</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_text_remind">
               <property name="styleSheet">
                <string notr="true">font: 75 11pt &quot;Arial&quot;;
color: rgb(0, 0, 255);</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_playbutton" native="true">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>40</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>40</number>
          </property>
          <property name="bottomMargin">
           <number>15</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_play">
            <property name="minimumSize">
             <size>
              <width>35</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>75</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="ccResources.qrc">
              <normaloff>:/icon/play.ico</normaloff>
              <normalon>:/icon/play.ico</normalon>:/icon/play.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoRepeat">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_pause">
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>75</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="ccResources.qrc">
              <normaloff>:/icon/pause.ico</normaloff>:/icon/pause.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_stop">
            <property name="minimumSize">
             <size>
              <width>35</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>75</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="ccResources.qrc">
              <normaloff>:/icon/stop.ico</normaloff>:/icon/stop.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_previous">
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>75</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="ccResources.qrc">
              <normaloff>:/icon/Previous.ico</normaloff>:/icon/Previous.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_next">
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>75</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="ccResources.qrc">
              <normaloff>:/icon/Fastforward.ico</normaloff>:/icon/Fastforward.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_cycle">
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>75</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="ccResources.qrc">
              <normaloff>:/icon/cycle.ico</normaloff>:/icon/cycle.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="ccResources.qrc"/>
 </resources>
 <connections/>
</ui>
